xml/art_slert_hw_testing.xml

```diff
@@ -176,95 +176,95 @@
     <title>Determining Latencies with the Hardware Latency Detector</title>
 
     <para> The <emphasis>hardware latency detector</emphasis>
-        (<filename>hwlat_detector.ko</filename>) is installed with the
-      real-time kernel (the <package>kernel-rt</package> package).
+        (<command>hwlatdetect</command>) is a program that controls the kernel
+      latency detector module (hwlat_detector.ko), installed with the
```

Comment:
```suggestion
      latency detector module (<filename>hwlat_detector.ko</filename>), installed with the
```

Reply:
Thank you for the suggestions! I integrated a bunch of them.